### PR TITLE
Change default behavior of capability collapsible card to be closed o…

### PR DESCRIPTION
**Pull request by uchiha-vivek**
**Change default behavior of capability collapsible card to be closed on Capabilities #37**

 **What was the issue **
1-) The capability collapsible card is opened by default
2-)  An empty capability is created on UI without clicking the “Add New” button.

Please Review it 
default
capability

### CODE REVIEW

File: ui/src/pages/ChatConfiguration/ChatConfiguration.jsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ const BotConfiguration = () => {
                 console.log({k :configs[0].inference[0]?.id})
```
Comment (sethumadhavan-k) [inline ui/src/pages/ChatConfiguration/ChatConfiguration.jsx:99]: remove this console log

File: ui/src/pages/ChatConfiguration/ChatConfiguration.jsx [hunk 2]
```diff
@@ -99,7 +99,7 @@ const BotConfiguration = () => {
...
 
                 if(configs[0].capabilities?.length == 0) {
-                    setCapabalities([{id: undefined, title:`Capability 1`, name:"", description:"", requirements: [], isCollapse: false}])
+                    setCapabalities([])
                 }else{
                     setCapabalities(configs[0].capabilities)
                 }
```
Comment (sethumadhavan-k) [inline ui/src/pages/ChatConfiguration/ChatConfiguration.jsx:105]: There's no need to initialize an empty array; instead, you can simply check if the `capabilities` array has a length greater than `0`.

**Comment by uchiha-vivek** [comment]
@sethumadhavan-k  yes sir Noted !

**Comment by uchiha-vivek** [comment]
@sethumadhavan-k  Please review

**Comment by sethumadhavan-k** [comment]
@uchiha-vivek remove the 'documents' dependencies from documents/package-lock.json and the 'raggenie' dependency from ui/package.json.

**Comment by uchiha-vivek** [comment]
@sethumadhavan-k ok

**Comment by uchiha-vivek** [comment]
> @uchiha-vivek remove the 'documents' dependencies from documents/package-lock.json and the 'raggenie' dependency from ui/package.json.

@sethumadhavan-k   I am removing it but its again coming by default,  i tried to remove it from both package-lock.json and then package.json.

**Comment by sethumadhavan-k** [comment]
@uchiha-vivek please remove those; they aren't needed anymore!

**Comment by sethumadhavan-k** [comment]
@uchiha-vivek  "Did you try running the commands `npm uninstall documents` and `npm uninstall raggenie`?"

**Comment by uchiha-vivek** [comment]
> @uchiha-vivek "Did you try running the commands `npm uninstall documents` and `npm uninstall raggenie`?"

@sethumadhavan-k  i did, i think i should again fork the repo once , let me try this once .

**Pull request merged**